tests/test_nxapi.rb

```diff
@@ -15,6 +15,9 @@
 # limitations under the License.
 
 require_relative 'basetest'
+require_relative '../lib/cisco_node_utils/node'
+
+include Cisco
```

Comment:
For the Node call here:

https://github.com/cisco/cisco-network-node-utils/pull/320/files#diff-02a78d01adf8c91e7e322c0263fb9f20R39

...but I can just add the module ref to the call to avoid the include. I'll add it.